src/Adapter/MSTest.CoreAdapter/ObjectModel/TestMethod.cs

```diff
@@ -57,8 +57,20 @@ public TestMethod(MethodBase method, string name, string fullClassName, string a
 
             ManagedNameHelper.GetManagedName(method, out var managedType, out var managedMethod);
 
-            this.ManagedType = managedType;
-            this.ManagedMethod = managedMethod;
+            // ManagedNameHelpers currently does not support spaces in method names.
+            // If there are spaces in the method name, we'll use the legacy way to find the method.
+            if (!managedMethod.Contains(" "))
```

Comment:
Do we need this check given `HasManagedMethodAndType` already has a similar check and the consumers already check that today?

Reply:
It’s  is a temporary workaround for this issue. https://github.com/microsoft/vstest/issues/2733

Reply:
ah, could we add that comment to the code?